hack/install-kubevirtci.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 
 organization=kubevirt
-commit="1c2922cd9ebaabf7006dc68aa8df70ad84d96d4b"
+commit="22a492a3de940bc1972bf66be13e836accc4533c"
```

Comment:
i think kubevirtci has new image for okd 4.1

db8c24bf830bb927f01829e6c9f083627fe6b832

Reply:
Thanks Or. I think this is on hold anyways, until we get multi-nic support in OKD.

Reply:
Yep, without multinic at okd4 we cannot test this.